### PR TITLE
CAV-392 - Gateway - Passcode persisting failure & passcode verificati…

### PR DESCRIPTION
**Pull request by johno1985**
…on check failure returns incorrect status code & message

 - set http timeout to 30 seconds

### CODE REVIEW

**Comment by johno1985** [comment]
rebuild

File: app/uk/gov/hmrc/cipphonenumber/connectors/VerifyConnector.scala [hunk 1]
```diff
@@ -39,19 +40,22 @@ class VerifyConnector @Inject()(httpClientV2: HttpClientV2, config: AppConfig)
   def verify(body: JsValue)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
     httpClientV2
       .post(url"$verifyUrl")
+      .transform(_.withRequestTimeout(30.seconds))
```
Comment (brdgeth) [inline app/uk/gov/hmrc/cipphonenumber/connectors/VerifyConnector.scala:43]: Again, we should make the timeouts configurable.

**Review by brdgeth — COMMENTED** [review]
config for timeouts please

File: app/uk/gov/hmrc/cipphonenumber/connectors/VerifyConnector.scala (hunk 1 above)
Comment (johno1985) [reply app/uk/gov/hmrc/cipphonenumber/connectors/VerifyConnector.scala:43]: Agreed, it will be done shortly

**Comment by johno1985** [comment]
rebuild

**Review by mar36tin — APPROVED** [review]

**Pull request merged**